unstable/wlr-data-control-unstable-v1.xml

```diff
@@ -129,6 +138,30 @@
         the client.
       </description>
     </event>
+
+    <!-- Version 2 additions -->
+
+    <event name="primary_selection" since="2">
+      <description summary="introduce a new wlr_data_control_offer">
+        The selection event is sent out to notify the client of a new
+        wlr_data_control_offer for the primary selection for this device. The
+        wlr_data_control_device.data_offer and the wlr_data_control_offer.offer
+        events are sent out before this event to introduce the data offer
+        object. The primary_selection event is sent to a client when a new
+        primary selection is set.
+      </description>
+      <arg name="id" type="object" interface="zwlr_data_control_offer_v1"
+        allow-null="true"/>
+    </event>
+
+    <request name="set_primary_selection" since="2">
+      <description summary="copy data to the primary selection">
+        All objects created by the device will still remain valid, until their
+        appropriate destroy request has been called.
```

Comment:
It would be nice to have a note about primary-selection support. Not all compositors support primary-selection, so we should make it clear that this request may be ignored by the compositor.

In the general case, even if primary-selection is supported, the compositor is free to ignore this request.

Additionally, it seems that the request description is a little bit off.